File: src/main/java/org/ventanas/Java/Ejercicio5.java
package org.ventanas.Java;
// 5. Tabla de multiplicar
// Pedir un número al usuario e imprimir su tabla de multiplicar del 1 al 10.
import java.util.Scanner;

public class Ejercicio5 {
    public static void execute() {
        Scanner scanner = new Scanner(System.in);
        System.out.print("Dame un número: ");
        try {
            int num = scanner.nextInt();
            for (int i = 1; i <= 10; i++) {
                System.out.println(i + " x " + num + " = " + (i * num));
            }
        } catch (Exception e) {
            System.out.println("Error al insertar el número, debe ser entero.");
        }
    }
}

File: src/main/java/org/ventanas/Kotlin/Ejercicio5.kt
package org.ventanas.Kotlin
//5. Tabla de multiplicar
// Pedir un número al usuario e imprimir su tabla de multiplicar del 1 al 10.

fun  ejercicio5() {
    print("Dame un número: ")
    try {
        val num = readln().toInt()

        for (i in 1..10){
            println("$i x $num = ${i*num}")
        }

    }catch (e:Exception){
        println("Error al insertar el numero, debe de ser entero.")
    }



}